abstract_specification.md

```diff
@@ -18,6 +18,15 @@ SOMA is an abstract _API specification_, with the goal of enabling multiple conc
 
 This document is attempting to codify the abstract, language-neutral SOMA data model and functional operations. Other specifications will document specific language bindings and particular storage system implementations. Where the term `language-specific SOMA specification` is used below, it implies a specification of the SOMA API as it is presented in a given language or computing environment (e.g., the SOMA Python API), common across all storage engine implementations in that language.
 
+# Lifecycle Stages
+
+The SOMA API uses [RStudio's lifecycle stage model](https://lifecycle.r-lib.org) to indicate the maturity of its classes and methods. The lifecycle stages are:
```

Comment:
Suggest:   ... of its classes, methods, parameters or interfaces.  

In other words, I think it needs to include method parameters, global state (if any), etc.